src/main/resources/db_scripts/db_script_v0.sql

```diff
@@ -0,0 +1,45 @@
+create table "workflow" (
+  "name" VARCHAR(45) NOT NULL UNIQUE,
+  "is_active" BOOLEAN NOT NULL,
+  "created" TIMESTAMP NOT NULL,
+  "updated" TIMESTAMP,
+  "id" BIGSERIAL NOT NULL PRIMARY KEY
+);
+
+create table "job_instance" (
+  "job_name" VARCHAR NOT NULL,
+  "job_definition_id" BIGINT NOT NULL,
+  "event_id" VARCHAR(70) NOT NULL UNIQUE,
+  "job_type" VARCHAR NOT NULL,
+  "variables" VARCHAR NOT NULL,
```

Comment:
I think `job_type` and `variable` can be null and are they needed? You have the `job_definition_id` and the `job_definition` has there

Reply:
job_type cant be null but variable can be. 
And about that job_definition_id, this is fine because it is reference to job_definition table.

Reply:
Another issue #28